src/Mailer/Email.php

```diff
@@ -492,7 +492,7 @@ public function getReadReceipt()
      * @return $this
```

Comment:
Shouldn't we now change the docblock too to `@return self` to match actual typehint?

Reply:
I think we dont have to. one is php language, the other for IDE to understand chaining.